plugins/field-colour-hsv-sliders/README.md

```diff
@@ -0,0 +1,72 @@
+# @blockly/field-colour-hsv-sliders [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+A [Blockly](https://www.npmjs.com/package/blockly) colour field using HSV sliders for editing.
+
+![A Blockly workspace showing the HSV sliders.](readme-media/hsv_sliders_screenshot.png)
+
+## Installation
+
+### Yarn
+```
+yarn add @blockly/field-colour-hsv-sliders
+```
+
+### npm
+```
+npm install @blockly/field-colour-hsv-sliders --save
+```
+
+## Usage
+This plugin adds a field type `FieldColourHsvSliders` that is registered to the name `'field_colour_hsv_sliders'`. This field is an extension of the Blockly.FieldColour field. See the [Blockly.FieldColour documentation](https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/colour#creation) on what parameters and configurations this field supports. However, unlike Blockly.FieldColour, FieldColourHsvSliders does not use colour swatches and thus ignores options related to the swatches such as `'colourOptions'`.
+
+To use it, you'll need to add this field to a block type definition, and add that block to your toolbox. See below for an example of defining a block type that uses this field.
+
```

Comment:
Can you add a comment that even though the UI is HSV the value is `'#rrggbb'`?

Reply:
Done.